services-custom/s3-transfer-manager/pom.xml

```diff
@@ -212,6 +212,17 @@
             <version>${commons-codec.verion}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.github.hakky54</groupId>
+            <artifactId>logcaptor</artifactId>
+            <version>2.8.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.6</version>
```

Comment:
Is version needed? Can't it inherit from parent pom?

Reply:
logcaptor plugin needs 2.x.x version, while parent uses 1.7.30. Upgrading to 2.x.x causes breaking changes

Reply:
Hmm, I see, then we can't use it unfortunately, because it will break customers. We could potentially making it a test dependency, but this could be error-prone since we could get confused and just remove the test scope in the future.

Reply:
got it, removed plugin and dependencies